mflowgen/Tile_PE/construct.py

```diff
@@ -224,6 +235,14 @@ def construct():
 
   g.update_params( parameters )
 
+  # Add custom timing scripts
+
+  for c_step in custom_timing_steps:
+    order = c_step.get_param( 'order' )
+    order.append( 'report-special-timing.tcl' )
+    c_step.set_param( 'order', order )
+    c_step.extend_postconditions( [{ 'pytest': 'inputs/test_timing.py' }] )
```

Comment:
Here is the postcondition on those three steps (to call the extra pytest functions)